plugins/source/aws/resources/services/cloudformation/stack_templates.go

```diff
@@ -29,6 +29,11 @@ func stackTemplates() *schema.Table {
 					PrimaryKey: true,
 				},
 			},
+			{
+				Name:     "template_body",
+				Type:     schema.TypeJSON,
```

Comment:
1 question: did you verify that the template will always be JSON?
Is there a possibility to see YAML, for instance?
https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_GetTemplate.html
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html

Possible workaround would be to parse as YAML always (JSON is a valid YAML) and then marshal as JSON

Reply:
It is always JSON. You can submit a template in YAML, but it is always returned to you in JSON